templates/web/angus/header.html

```diff
@@ -0,0 +1,77 @@
+[%
+    SET html_att = ' lang="' _ lang_code _ '"';
+    # For a right-to-left language, use the following line in your own header:
+    # SET html_att = html_att _ ' dir="rtl"';
+-%]
+<!doctype html>
+<!--[if lt IE 7]><html class="no-js ie6 iel8"[% html_att %]><![endif]-->
+<!--[if IE 7]>   <html class="no-js ie7 iel8"[% html_att %]><![endif]-->
+<!--[if IE 8]>   <html class="no-js ie8 iel8"[% html_att %]><![endif]-->
+<!--[if IE 9]>   <html class="no-js ie9"[% html_att %]><![endif]-->
+<!--[if gt IE 9]><!--><html class="no-js"[% html_att %]><!--<![endif]-->
+    <head>
+        <meta name="viewport" content="initial-scale=1.0">
+
+        <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
+        <meta name="HandHeldFriendly" content="true">
+        <meta name="mobileoptimized" content="0">
+
+        [% INCLUDE 'header_opengraph.html' %]
+        [% SET start = c.config.ADMIN_BASE_URL IF admin;
+
+           # For clarity, the 'fixmystreet' moniker (for fixmystreet.com) puts
+           # it stylesheets under fixmystreet.com
+           IF c.cobrand.moniker == 'fixmystreet';
```

Comment:
This can also be dropped :)
I guess adding more hooks to the parent header so this doesn't need to all be overridden is probably not worth it.